logging package name conflict after generation

<!--
Thank you for helping to improve Terrajet!

Please be sure to search for open issues before raising a new one. We use issues
for bug reports and feature requests. Please find us at https://slack.crossplane.io
for questions, support, and discussion.
-->

### What happened?
I encountered `logging` package conflict during the generation of https://github.com/AbsaOSS/provider-jet-rancher
```
17:27:05 [ .. ] golangci-lint
WARN [runner] Can't run linter goanalysis_metalinter: bodyclose: failed prerequisites: [buildssa@github.com/crossplane-contrib/provider-jet-rancher/internal/controller: analysis skipped: errors in package: [/Users/xxx/absa/provider-jet-rancher/internal/controller/zz_setup.go:43:2: logging redeclared in this block /Users/xxx/absa/provider-jet-rancher/internal/controller/zz_setup.go:23:2: 	other declaration of logging /Users/xxx/absa/provider-jet-rancher/internal/controller/zz_setup.go:101:11: Setup not declared by package logging]]
WARN [runner] Can't run linter unused: buildir: analysis skipped: errors in package: [/Users/xxx/absa/provider-jet-rancher/internal/controller/zz_setup.go:43:2: logging redeclared in this block /Users/xxx/absa/provider-jet-rancher/internal/controller/zz_setup.go:23:2: 	other declaration of logging /Users/xxx/absa/provider-jet-rancher/internal/controller/zz_setup.go:101:11: Setup not declared by package logging]
ERRO Running error: buildir: analysis skipped: errors in package: [/Users/xxx/absa/provider-jet-rancher/internal/controller/zz_setup.go:43:2: logging redeclared in this block /Users/xxx/absa/provider-jet-rancher/internal/controller/zz_setup.go:23:2: 	other declaration of logging /Users/xxx/absa/provider-jet-rancher/internal/controller/zz_setup.go:101:11: Setup not declared by package logging]
17:27:17 [FAIL]
```
In the `internal/controller/zz_setup.go` we can observe two conflicting `logging` package imports

```
import (
...
    "github.com/crossplane/crossplane-runtime/pkg/logging"
...
    logging "github.com/crossplane-contrib/provider-jet-rancher/internal/controller/cluster/logging"
```

### How can we reproduce it?

Clone https://github.com/AbsaOSS/provider-jet-rancher , `make generate`

--- a/go.mod
+++ b/go.mod
@@ -2,12 +2,12 @@ module github.com/crossplane-contrib/provider-jet-rancher
 
 go 1.16
 
-replace github.com/crossplane/terrajet => github.com/turkenh/terrajet v0.0.0-20220114225802-9f11209e5cac
+replace github.com/crossplane/terrajet => github.com/turkenh/terrajet v0.0.0-20220125090152-40fd9a14dc4d
 
 require (
 	github.com/crossplane/crossplane-runtime v0.15.1-0.20211004150827-579c1833b513
 	github.com/crossplane/crossplane-tools v0.0.0-20210916125540-071de511ae8e
-	github.com/crossplane/terrajet v0.3.1
+	github.com/crossplane/terrajet v0.3.2
 	github.com/hashicorp/terraform-plugin-sdk/v2 v2.7.0
 	github.com/pkg/errors v0.9.1
 	github.com/rancher/terraform-provider-rancher2 v1.22.1

